### DISCUSSION TITLE
With Emporia Vue Zigbee HAN - what are those different _value?

### DISCUSSION THREAD

**Discussion by luisfocosta** [jertel/vuegraf#93] [Q&A] [answered]
I have seen in my Vue bucket (InfluxDB 2x).
The table has a field "Detailed" with 2 possible values: FALSE|TRUE. Usage datapoints (in the vicinity) are off by a few watts.
E.g.:
table	_start	_stop	_time	_value	_field	detailed
0	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T03:23:30Z	728.329216	usage	FALSE
0	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T03:24:30Z	726.2000392	usage	FALSE
0	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T03:25:30Z	717.6750298	usage	FALSE
0	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T03:26:30Z	724.1541843	usage	FALSE
0	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T03:27:30Z	724.2292089	usage	FALSE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:27:50Z	1273.600032	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:00Z	1284.000033	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:10Z	1286.400054	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:20Z	1294.20007	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:30Z	1311.000114	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:40Z	1366.200112	usage	TRUE
1	2022-05-12T02:27:45.509070552Z	2022-05-12T03:27:45.509070552Z	2022-05-12T02:28:50Z	1375.200109	usage	TRUE

What values should I consider? The ones where detailed=FALSE? The entries where detailed=TRUE are fewer and not up-to-date
BTW I have solar production (measured another way)
What do you think?

**Comment by jertel** [comment, marked as answer]
This discussion should help: https://github.com/jertel/vuegraf/discussions/57
